src/main/kotlin/com/ampnet/headlesscmsservice/enums/MailType.kt

```diff
@@ -102,3 +102,5 @@ enum class MailType(val id: Int, val defaultTemplateKey: String, val defaultTitl
 
     abstract fun getRequiredFields(): List<MailFieldName>
 }
+
+fun MailType.name() = this.name.toLowerCase()
```

Comment:
So `MailType` already has a field called `name` and now you are adding a method called `name()` which will provide the `name` in lower case? To me this seems like a thing where it is easy to make a mistake and use `name` instead of `name()`. My suggestion is to rename this method to `lowerCaseName()`. You can even make it a `val` with a getter then.